confd/service/service_test.go

```diff
@@ -61,3 +67,42 @@ func TestCreateServiceWithNonStringNameKey(t *testing.T) {
 	service := createService(raw)
 	require.Nil(t, service)
 }
+
```

Comment:
We should create a `TestCreateService` with multiple `t.Run("Create service ...)`

Reply:
I have created a `TestCreateService` method with multiple `t.Run`s